decidim-accountability/app/views/decidim/accountability/admin/timeline_entries/index.html.erb

```diff
@@ -1,8 +1,10 @@
 <div class="card">
   <div class="item_show__header">
     <h2 class="item_show__header-title">
-      <%= link_to "#{translated_attribute(result.title)} > ", edit_result_path(result) %>
-      <%= t(".title") %>
+      <div>
```

Comment:
I don't like this lone div without classes, and as far as I could see, this is the only case where we need it for the `h2.item_show__header-title`. Maybe can we find an alternative so we don't introduce that?

Reply:
As far as i can see, the `h2.item_show__header-title` has a `justify-content: space-between;`. If we remove that we could align everything to the left ( including buttons). To make it work nice, we need some extra container there. 

I could use a span for this kind of situations ( there is more than one place where we need it).

Reply:
Ok, I don't like how this looks like but I understand that the alternatives aren't so easy, so we'd leave it like it is for now